src/project_model/package_dependency.rs

```diff
@@ -28,8 +36,17 @@ impl PackageDependency {
             })
     }
 
+    pub fn map_from_metadata_dependencies(
```

Comment:
Zmienilbym nazwe na metadata_dep_to_package_dep

Reply:
bez sensu trochę, bo to nazwa metody i jest wołane PackageDependency::map_from_metadata_dependencies(...)